Slayer Plugin: Retrieve Reward Points via Varbits

xDemoN opened the pull request:
This is a recommit of #2933 

@deathbeam showed interest, but the author abandoned it

Adam- commented:
Does this work? I wouldn't think the varbit would be always available.

deathbeam commented:
As long as its used only for getting points from slayer rewards interface then yeah (as its available only there afaik)

xDemoN commented:
I pulled this from the previous PR, rebased and updated it but whatever I've done doesn't work. I don't have enough knowledge to fix it either so I'm closing it.
